lisa/transformers/dom0_kernel_installer.py

```diff
@@ -187,28 +172,28 @@ def _copy_kernel_binary(
     )
 
 
-def _update_linux_loader(
+def _update_mariner_config(
     node: Node,
     is_initrd: bool,
     current_kernel: str,
     new_kernel: str,
 ) -> None:
-    ll_conf_file: str = "/boot/efi/linuxloader.conf"
+    ll_conf_file: str = "/boot/mariner.cfg"
     sed = node.tools[Sed]
 
     # Modify the linuxloader.conf to point new kernel binary
```

Comment:
Please update this comment accordingly.

Reply:
@pupacha Thnx for pointing this. It is done. Please check now